### PR TITLE
Modify some code of Sym.py

### PR DESCRIPTION
**Pull request by Justin091136**
Modify some code of Sym.py

### CODE REVIEW

File: w2/num.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
-class NUM:
+class Num:
```
Comment (Aakarsh-Sathish) [inline w2/num.py]: Should it be NUM as discussed?

File: w2/num.py [hunk 2]
```diff
@@ -25,15 +25,17 @@ def mid(self):
...
-
     def norm(self, x):
         return x if x == "?" else (x - self.lo) / (self.hi - self.lo + 1E-30)
 
+    """
+    # Currently not supported
+    def small(self):
+        return the["cohen"] * self.div()
+
     def like(self, x, _):
```
Comment (Aakarsh-Sathish) [inline w2/num.py:36]: Should the Like be implemented? Prof said no right?

File: w2/num.py (hunk 1 above)
Comment (Justin091136) [reply w2/num.py]: Yes. I will update it with the next commit.

File: w2/sym.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
-class SYM:
+import math
+
+class Sym:
```
Comment (Aakarsh-Sathish) [inline w2/sym.py]: SYM?

File: w2/num.py (hunk 2 above)
Comment (Justin091136) [reply w2/num.py:36]: We don't need to implement it for this weeks assignment. So I just comment it out.

File: w2/sym.py (hunk 3 above)
Comment (Justin091136) [reply w2/sym.py]: Yeah. I will update it with the next commit.

**Review by Aakarsh-Sathish — APPROVED** [review]
Looks good!

**Pull request merged**